database/frontend/react.json

```
@@ -1,10 +1,18 @@
 [
     {
-        "name":"React js",
-        "description":"Free online course for learning react js",
-        "url":"https://scrimba.com/learn/learnreact#",
-        "category":"frontend",
-        "subcategory":"react",
-        "language":"english"
+        "name": "React js",
+        "description": "Free online course for learning react js",
+        "url": "https://scrimba.com/learn/learnreact#",
+        "category": "frontend",
+        "subcategory": "react",
+        "language": "english"
+    },
+    {
+        "name": "Epic React",
+        "description": "Epic React is a paid comprehensive workshop series by Kent C. Dodds that equips intermediate to advanced web developers with the knowledge and skills to build robust, well-tested React applications that scale.",

```

[Nitpick] This sounds like an interesting series, but we’re trying to only showcase free resources. Consider replacing this series with a free one.
```suggestion
        "description": "Epic React is a paid comprehensive workshop series by Kent C. Dodds that equips intermediate to advanced web developers with the knowledge and skills to build robust, well-tested React applications that scale.",

```